src/pages/Checkout/Checkout.scss

```
@@ -1 +1,125 @@
-//q
+@import '../../styles/mixin';
+@import '../../styles/variables';
+
+.checkout {
+  @include textStyle(1em, 500);
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  padding: 40px;
+
+  .article-checkout {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    width: 60%;
+    padding: 40px;
+
+    .checkout-article-h1 {
+      margin-bottom: 2em;
+      font-size: 1.7em;
+    }
+
+    .section-delivery-infomation {
+      display: flex;
+      flex-direction: column;
+
+      .user-information-h2 {
+        margin-bottom: 0.7em;
+        font-size: 1.4em;
+      }
+
+      .user-information-values {
+        display: flex;
+        flex-direction: column;
+        font-size: 0.9em;
+
+        .user-name-value {
+          margin: 0.4em 0em;
+        }
+
+        .user-address-value {
+          margin: 0em 0em 0.4em 0em;
+        }
+
+        .user-phonenumber-value {
+          margin: 0em 0em 0.4em 0em;
+        }
```

```suggestion
        .user-address-value {
          margin-botton: 0.4em;
        }

        .user-phonenumber-value {
          margin-botton: 0.4em;
        }
```
특정 위치만 스타일을 적용한다면 위와 같이 적용할 수 있겠습니다!